File: launchdarkly/php/src/DeleteFlagFollowerExample.php
<?php

namespace OSEG\LaunchDarklyExamples;

require_once __DIR__ . '/../vendor/autoload.php';

use SplFileObject;
use LaunchDarkly;

$config = LaunchDarkly\Client\Configuration::getDefaultConfiguration();
$config->setApiKey("Authorization", "YOUR_API_KEY");

try {
    (new LaunchDarkly\Client\Api\FollowFlagsApi(config: $config))->deleteFlagFollower(
        project_key: null,
        feature_flag_key: null,
        environment_key: null,
        member_id: null,
    );
} catch (LaunchDarkly\Client\ApiException $e) {
    echo "Exception when calling FollowFlagsApi#deleteFlagFollower: {$e->getMessage()}";
}

File: launchdarkly/ruby/src/DeleteFlagFollowerExample.rb
require "json"
require "launchdarkly_client"

LaunchDarklyClient.configure do |config|
    config.api_key["Authorization"] = "YOUR_API_KEY"
end

begin
    LaunchDarklyClient::FollowFlagsApi.new.delete_flag_follower(
        "projectKey_string", # project_key
        "featureFlagKey_string", # feature_flag_key
        "environmentKey_string", # environment_key
        "memberId_string", # member_id
    )
rescue LaunchDarklyClient::ApiError => e
    puts "Exception when calling FollowFlagsApi#delete_flag_follower: #{e}"
end
